molo/core/tests/test_middleware.py

```diff
@@ -77,10 +88,38 @@ def test_ga_submit_tracking_with_custom_params(self, mock_method):
 
         middleware = MoloGoogleAnalyticsMiddleware()
         account = ''
+        middleware.submit_tracking(
+            account, request, response, custom_params)
+
+        self.assertTrue('custom_params' in mock_method._mock_call_args[1])
+        self.assertFalse('user_id' in mock_method._mock_call_args[1])
+        self.assertTrue(mock_method._mock_call_args[1]['custom_params'],
+                        custom_params)
+
+    @patch('molo.core.middleware.build_ga_params')
+    def test_ga_submit_tracking_with_custom_params__authenticated(
```

Comment:
double underscore here?

Reply:
it's just to note that this test's only difference from the one above is that a user is authenticated.